### ISSUE TITLE
Voodoo Doll nerf

### ISSUE THREAD

**Issue by WightofShadows** [TheCyberBrick/The-Betweenlands#645]
A little gripe I've had for a little while. Voodoo Dolls can be used up to 24 times each and dish out tons of damage to literally anything nearby. And although they're considered "rare loot" it isn't that hard to get an ample supply of them through Item Scrolls. They kind of break the strategy built around fighting mobs. Someone in the Discord also pointed out that they can be applied with Mending, making them essentially infinite.

Simply reducing their durability (possibly even making them one-use), and/or giving them a cooldown, would make them more fair while still keeping them valuable items for clearing crowds of weaker mobs. Taking it a step further their rarity from Item Scrolls could also be increased, although that may not be necessary with the previous nerf in account. If this does end up making them less valuable then even increasing the damage they deal to make up for it would be fine.

Another thing: They used to not work on Wights. I think it'd be good if they worked like that again.

**Comment by Divineaspect** [comment]
I'll chime in to support giving them a cooldown.

**Comment by 63502** [comment]
I'll say yes to a nerf for them too. 

Increasing the damage output by a lot feels a bit counterintuitive, but it could work if the doll has a very long universal cooldown(20-30 seconds across all dolls a player possesses). It would be just as valuable, but more as an emergency nope switch rather than the current weapon of mindless slaughter.

**Comment by PetasiGit** [comment]
Perhaps remove of the ability to put books on them whatsoever and add a 5 second cool-down between uses, sort of like how vanilla enderpearls are?

**Comment by TripleHeadedSheep** [comment]
I agree with nerfing them. Perhaps they deal a little bit of damage to the player as well? sort of a blood trade if you will. That way you can't just spam them and you really have to consider when to use them.
